lizard_auth_client/tests.py

```diff
@@ -390,6 +404,13 @@ def test_build_sso_portal_action_url2(self):
         self.assertTrue(views.build_sso_portal_action_url('something',
                                                           domain='ab.cd'))
 
+    def test_login(self):
```

Comment:
In hoeverre heb je de daadwerkelijke middleware getest? Met het mocken van de `process_request()` zou je makkelijk in een paar regels de twee scenario's (wel/niet ingelogd op de SSO) kunnen testen.